docs/polygon_draw.js

```diff
@@ -448,10 +447,12 @@ function createHelpIcon(url) {
  * @param {google.maps.Map} map Map to display regions on
  * @param {Promise<any>} firebasePromise Promise with Firebase damage data (also
  *     implies that authentication is complete)
+ * @param {Promise<any>} userShapesPromise Promise with user shapes data
```

Comment:
nit: can type this more strictly, especially after making my proposed change to firestore_document